tests/data_context/cloud_data_context/test_checkpoint_crud.py

```diff
@@ -184,6 +183,7 @@ def test_cloud_backed_data_context_add_checkpoint(
         )
 
         mock_post.assert_called_with(
+            mock.ANY,  # requests.Session object
```

Comment:
Trying to catch up. Why are using `mock.ANY` instead of the old `shared_called_with_request_kwargs` fixture?
Did we decide this was over-testing?